cmd/root.go

```diff
@@ -204,27 +232,64 @@ func (ci *codeInj) Parse() *bf.Node {
 			ci.extensions = nil
 		}
 	}
-	ci.mdExt = bf.FencedCode | bf.Tables | bf.HeadingIDs | bf.NoIntraEmphasis
-	return bf.New(bf.WithExtensions(ci.mdExt)).Parse(f)
+	return markdown.NewParser().Parse(text.NewReader(f)), nil
 }
 
-func (ci *codeInj) Inject(n *bf.Node, entering bool) bf.WalkStatus {
+// snippet is split to more siblings
+func (ci *codeInj) IsMatchWithSiblings(n ast.Node) (bool, []byte, []ast.Node) {
+	//tnode, is := n.(*ast.Text)
+	//if !is {
+	//	return false, nil, nil
+	//}
+	//literal1 := tnode.Segment.Value(ci.originSource)
```

Comment:
Some commented code still included.  Can we clean that out?

Reply:
thx for catching this, will remove